feat(changelog): jobs-added-serverless-jobs-console-improvements 2025-01-13 (#4209)
scaleway/docs-content: changelog/january2025/2025-01-13-jobs-added-serverless-jobs-console-improvements.mdx

---
title: Serverless Jobs console improvements
status: added
author:
    fullname: 'Join the #serverless-jobs channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2025-01-13
category: serverless
product: jobs
---

> - Added easy access to unique identifier of ressources in Job Definition view

Reviewer: ```suggestion
- Added easy access to the unique identifier of resources in the **Job definition** panel
```

> - Added direct access to documentation in listing of Job Definitions

Reviewer: ```suggestion
- Added direct access to documentation in Job listing
```

> - Added direct access to documentation in listing of Job Definitions

Reviewer: @SamyOubouaziz can you confirm this is right? Looking at the console this is what it seems to be to me

> - Improved error messages display for Job Runs

Reviewer: ```suggestion
- Improved error message display for Job runs
```
